### PR TITLE
Starting from a clean slate

### PR DESCRIPTION
**Pull request by msal4**
## Description
Rewrote the app
## How Has This Been Tested?

`jest --detectOpenHandles`
## Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [x] Breaking change (fixes, design refactors, or anything that could cause existing functionality to change)

## Checklist:

* [x] My code follows the code style of this project.
* [x] My change required a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [x] I have added tests to cover my changes.
* [x] All new and existing tests passed.
* [x] I have ~~emulated in~~ **used a real** Android  **device** and tested my change.
* [ ] I have emulated in iOS and tested my change.

### CODE REVIEW

File: android/app/src/main/java/com/jellyfin/MainActivity.java
Comment (jonathan-irvin) [inline android/app/src/main/java/com/jellyfin/MainActivity.java]: Can you explain the naming convention?

File: package.json
Comment (jonathan-irvin) [inline package.json:12]: Why was this removed?

File: src/firebase-app.js
Comment (jonathan-irvin) [inline src/firebase-app.js:14]: These are your configs, do they need to be committed?

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (jonathan-irvin) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Why was this deleted?

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (jonathan-irvin) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Why was this deleted?

File: .editorconfig
Comment (jonathan-irvin) [inline .editorconfig]: Why was this deleted?

File: LICENSE
Comment (jonathan-irvin) [inline LICENSE]: Why was this deleted?

File: PULL_REQUEST_TEMPLATE.md
Comment (jonathan-irvin) [inline PULL_REQUEST_TEMPLATE.md]: Why was this deleted? Please add this back under the `.github` folder.

File: ios/Podfile
Comment (jonathan-irvin) [inline ios/Podfile:1]: Won't we need this?

File: package.json
Comment (jonathan-irvin) [inline package.json:15]: Please add this back

File: package.json
Comment (jonathan-irvin) [inline package.json:16]: Please add this back

File: package.json
Comment (jonathan-irvin) [inline package.json]: Please add this back

File: package.json
Comment (jonathan-irvin) [inline package.json]: Change this back to jelly-fin

File: android/app/src/main/java/com/jellyfin/MainActivity.java
Comment (msal4) [reply android/app/src/main/java/com/jellyfin/MainActivity.java]: Its not a naming convention. I deleted app.json, and then recreated it, thats why the name is different. Do you want to use camel casing?

**Review by jonathan-irvin — CHANGES_REQUESTED** [review]
Great job cleaning the slate and getting our build to pass.  I've left several comments and I'd also like @emmanuelvr to weigh in on this PR as well before we merge it.

File: android/app/src/main/java/com/jellyfin/MainActivity.java
Comment (jonathan-irvin) [reply android/app/src/main/java/com/jellyfin/MainActivity.java]: Yes please 👍

File: src/firebase-app.js
Comment (msal4) [reply src/firebase-app.js:14]: it's a config for a test account so that the build doesn't fail, the contributors have the option to make a firebase-config.json and place their own credentials. I also added it to the .gitignore file.

File: ios/Podfile
Comment (msal4) [reply ios/Podfile:1]: No, it's just ios firebase dependencies, which we won't anymore

File: src/firebase-app.js
Comment (jonathan-irvin) [reply src/firebase-app.js:14]: Cool, thanks!

**Review by jonathan-irvin — APPROVED** [review]
It looks good, I'm going to hold off merging until @emmanuelvr takes a look

**Pull request merged**
